examples/cli_example_no_option.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of DoMapping.
+# Copyright (C) 2015, 2016 CERN.
+#
+# DoMapping is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# DoMapping is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with DoMapping; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+
+"""In this example, there is no option set, so by default
+"--ignore_index" option is enabled.
+Thus array indexes are ignored and for each array field, all items have to be
+of the same type.
+
+Run this example:
+.. code-block:: console
+    $ cd examples
+    $ python app.py
+The same result could be created with the cli:
+.. code-block:: console
+    $ doschema file1.json file2.json
+"""
+
+import json
+import os
+from io import open
+
+import chardet
+import click
+
+import doschema.validation
+
+schemas = ['jsonschema_f1.json', 'jsonschema_f2.json']
+
+schema_validator = doschema.validation.JSONSchemaValidator()
+for schema in schemas:
+    path = os.path.join(os.getcwd(), 'tests', 'data_files', schema)
```

Comment:
It would be better to have example files in the example directory and not use anything from the tests directory.

It would be great if the test JSON-Schema files could show the difference between `--with_index` and `--ignore-index`.